### ISSUE TITLE
PSNR:Nan

### ISSUE THREAD

**Issue by qiufengyuzhi** [thstkdgus35/EDSR-PyTorch#364]
Excuse me, I referred to the above conversation to modify my code, but still haven't solved the problem of the PSNR value being nan. I think the model still hasn't read the dataset. I would be very grateful if there have any suggestions about my problem.
![image](https://github.com/user-attachments/assets/5634f33f-620a-4132-972d-5afbde344cf2)

python main.py --n_GPUs 1 --rgb_range 1 --chunk_size 144 --n_hashes 4 --save_models --lr 1e-4 --decay 200-400-600-800 --epochs 1000 --chop --save_results --n_resblocks 32 --n_feats 256 --res_scale 0.1 --batch_size 16 --model NLSN --scale 2 --patch_size 96 --save NLSN_x2 --data_train DIV2K
![image](https://github.com/user-attachments/assets/ca6c22a1-c8d4-4e8a-b311-c68c54fcb2c8)

**Comment by doubleup626** [comment]
Have you solved this problem? I had the same problem

**Comment by xueyuan123456789** [comment]
![Image](https://github.com/user-attachments/assets/d5e9348a-dec5-49ca-832e-7a63bd60d50d)。Copy the validation set into the training set, and modify the parameters in option.py.
The data splitting is controlled by the data_range parameter, where the first 800 images in the training dataset folder are used for training, and images 801 to 900 are used for validation.

![Image](https://github.com/user-attachments/assets/c4d2d270-5222-4b9d-baa0-75883e77ee2b)

**Comment by xueyuan123456789** [comment]
![Image](https://github.com/user-attachments/assets/b1cbe625-3832-4e1c-9b89-dd360b8a4fb9)
